lib/msal-core/src/ScopeSet.ts

```diff
@@ -160,4 +153,71 @@ export class ScopeSet {
 
     // #endregion
 
+    /**
+     * @ignore
+     * Returns true if the scopes array only contains openid and/or profile
+     */
+    static onlyContainsOidcScopes(scopes: Array<string>): boolean {
+        let oidcScopesFound = 0;
+
+        if (scopes.indexOf(Constants.openidScope) > -1) {
+            oidcScopesFound += 1;
+        }
+
+        if (scopes.indexOf(Constants.profileScope) > -1) {
+            oidcScopesFound += 1;
+        }
+
+        const nonOidcScopes = scopes.length - oidcScopesFound;
+
+        return (nonOidcScopes === 0);
```

Comment:
```suggestion
        return (scopes.length === oidcScopesFound);
```

Reply:
I removed the unnecessary subtraction per your suggestion, but I also added a non-zero length check for the array (technically empty arrays shouldn't make it this far, but you can never be too careful).